library/ecp.c

```diff
@@ -245,6 +245,10 @@ const mbedtls_ecp_curve_info *mbedtls_ecp_curve_info_from_name( const char *name
 {
     const mbedtls_ecp_curve_info *curve_info;
 
+    /* Validate input parameter */
+    if ( name == NULL )
```

Comment:
This needs to be adapted to the Mbed TLS spacing style: `if( name == NULL ) ...` and `return( NULL );`